src/app/modules/shared_modules/keycard_popup/internal/wrong_pin_state.nim

```diff
@@ -44,7 +48,8 @@ method executeCancelCommand*(self: WrongPinState, controller: Controller) =
     self.flowType == FlowType.RenameKeycard or
     self.flowType == FlowType.ChangeKeycardPin or
     self.flowType == FlowType.ChangeKeycardPuk or
-    self.flowType == FlowType.ChangePairingCode:
+    self.flowType == FlowType.ChangePairingCode or
+    self.flowType == FlowType.CreateCopyOfAKeycard:
```

Comment:
Those conditions repeat several times. Can you refactor them to helper func?
```
   self.flowType == FlowType.RenameKeycard or
    self.flowType == FlowType.ChangeKeycardPin or
    self.flowType == FlowType.ChangeKeycardPuk or
    self.flowType == FlowType.ChangePairingCode or
    self.flowType == FlowType.CreateCopyOfAKeycard:
```

Reply:
@MishkaRogachev yes these conditions repeats, but I deliberately keep then, even they are the same, separately for each state. For example in state1  we're handling action1 the same for flows 1,2,3,4,5, but in case of state2 we're handling that way only flows 1,3 or 2,4,5... what I want to say is that the way how we need to handle appropriate action depends a lot on a certain state and flow and I found it easier for maintaining to keep it separately.

Was thinking about merging stuff or create handler function for each flow, not talking about this particular case but in general cause for each state we have `if` which separates handling for a certain flow, but for now found it easier for maintaining this way, but will see at the end.